docs/content/how-to/aws/create-aws-hosted-cluster-autonode.md

```diff
@@ -0,0 +1,452 @@
+---
+title: Create AutoNode hosted clusters powered by Karpenter
+---
+
+HyperShift AutoNode (Powered by Karpenter) is ...(TODO)
```

Comment:
is a feature that runs Karpenter management side as a control plane component while it watches ec2NodeClasses.karpenter.k8s.aws, nodePools.karpenter.sh and nodeclaims.karpenter.sh resource in the guest cluster.